src/Components/News/Detail/NewsDetail.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react';
+
+import { Typography, Box } from '@mui/material';
+import Title from '../../Title/Title';
+import style from '../../../Styles/NewsDetail/NewsDetail.module.css';
+
+const NewsDetail = () => {
+  const [newsDetail, setNewsDetail] = useState({
+    name: '',
+    content: '',
+    image: '',
+  });
+
+  const getNewsDetails = () => ({
+    name: 'Titulo de la noticia',
```

Comment:
the title should be received on props (I like this but is required by the ticket)